src/lib.rs

```diff
@@ -1,11 +1,13 @@
 use rocket::serde::json::{Json};
-use rocket::{Build, catch, catchers, Request, Rocket};
+use rocket::{Build, catch, catchers, Request, Rocket, routes};
 use rocket::State;
 
 mod config;
 mod responses;
+mod location;
 
 use responses::{FailResponse, OutpackError, OutpackSuccess};
+pub use self::location::LocationEntry;
```

Comment:
Why does this need to be `pub use` and not `use`?

Reply:
Looks vestigial, probably was because I was referencing that type in integration tests, but am not any longer so can remove this whole statement :+1: